pkg/node/provisioners/utilwrappers/fs_operations.go

```diff
@@ -96,8 +96,37 @@ func (fsOp *FSOperationsImpl) PrepareAndPerformMount(src, dst string, bindMount,
 	}
 	if err := fsOp.Mount(src, dst, opts); err != nil {
 		if wasCreated {
-			_ = fsOp.RmDir(dst)
+			err = fsOp.RmDir(dst)
```

Comment:
Why do we need err here, if we don't check it or use it?

Reply:
fixed